.travis.yml

```diff
@@ -29,9 +29,9 @@ before_script:
   - if [ $ES_COMPOSER_NODEV == "no" ] && ! $(phpenv version-name | grep -q '5.3'); then sudo composer require "guzzlehttp/guzzle" "4.2.*" --dev --no-ansi --no-progress --no-interaction; fi
 
 script:
+  - phpunit -c test/ --coverage-clover build/coverage/clover-unit.xml --group unit
```

Comment:
I see, haven’t thought of that. The best thing would be to only install and boot up elasticsearch after the unit tests were run. Perhaps we can set environment variables for each build and for the unit tests set the ES_HOST to null.

> On 21 May 2015, at 14:31, Igor Denisenko <notifications@github.com> wrote:
> 
> In .travis.yml <https://github.com/ruflin/Elastica/pull/858#discussion_r30796730>:
> 
> > @@ -29,9 +29,9 @@ before_script:
> >    - if [ $ES_COMPOSER_NODEV == "no" ] && ! $(phpenv version-name | grep -q '5.3'); then sudo composer require "guzzlehttp/guzzle" "4.2.*" --dev --no-ansi --no-progress --no-interaction; fi
> >  
> >  script:
> > +  - phpunit -c test/ --coverage-clover build/coverage/clover-unit.xml --group unit
> Just FYI, if unit tests will be placed before shutdown, they potentially can hit ES server, that obviously not thing unit test is supposed to do.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ruflin/Elastica/pull/858/files#r30796730>.
>